core/src/main/java/io/seata/core/rpc/processor/client/RmBranchRollbackProcessor.java

```diff
@@ -55,19 +55,19 @@ public void process(ChannelHandlerContext ctx, RpcMessage rpcMessage) throws Exc
         if (LOGGER.isInfoEnabled()) {
             LOGGER.info("rm handle branch rollback process:" + msg);
         }
-        handleBranchRollback(rpcMessage, remoteAddress, (BranchRollbackRequest) msg);
+        handleUndoLogDelete(rpcMessage, remoteAddress, (BranchRollbackRequest) msg);
```

Comment:
Why change the name to handleUndoLogDelete?

Reply:
done